client/src/templates/Challenges/classic/editor-tabs.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import { sortChallengeFiles } from '../../../../../utils/sort-challengefiles';
+import { ChallengeFile, ChallengeFiles } from '../../../redux/prop-types';
+import {
+  toggleVisibleEditor,
+  visibleEditorsSelector,
+  challengeFilesSelector
+} from '../redux';
+
+type VisibleEditorsType = {
```

Comment:
Some devs do not like being explicit (👀 @ojeytonwilliams 👀 )

```suggestion
type VisibleEditors = {
```